doc/source/conf.py

```diff
@@ -30,11 +36,16 @@ def get_root_dir():
 
 def get_igraphdir():
     '''Get igraph folder'''
+    # Python version
     vmaj, vmin = sys.version_info[:2]
-    root_folder = get_root_dir()
+
+    if is_inside_rtd:
+        venv_folder = f'/home/docs/checkouts/readthedocs.org/user_builds/igraph/envs/{rtd_version}/'
```

Comment:
How stable is this path? Is this documented and guaranteed to look like this in readthedocs, or have you just inferred this?

Reply:
Reverse engineering... They don't seem to give guarantees and call all their executables via full path...

Reply:
Could we "detect" it somehow from inside `conf.py`, e.g., with `os.getcwd()` so it won't break if they decide to change the path later?

Or, maybe with inspecting `__file__` from inside `conf.py` and chopping off components from the end if needed?

Reply:
that's the thing: they keep the venvs separate from the source tree so that'd be equally messy I think. Ideally we don't need these paths at all and we can use `import igraph` to get what we need, however last I checked pydoctor was unhappy about it (a bug in pydoctor I believe). Happy to give it another go.

Otherwise we could import a dummy package (e.g. sphinx itself) and get its `__file__` and use that to reconstruct the venv location

Reply:
> we could import a dummy package (e.g. sphinx itself) and get its __file__ and use that to reconstruct the venv location

That sounds like a good idea! Let's try it and I'm happy to merge this PR afterwards if we managed to sort this out (or if we concluded that it's not worth it).

Reply:
`import importlib + <black magic> = 42`

elegant solution found